apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue

```diff
@@ -95,13 +96,15 @@ const drawChart = () => {
         return text;
     });
 
-    props.legends.forEach((l) => {
-        const series = chartHelper.createXYColumnSeries(chart, {
-            name: l.label as string,
-            valueYField: l.name,
+    props.legends.forEach((legend) => {
+        const seriesSettings: Partial<am5xy.IXYSeriesSettings> = {
+            name: legend.label as string,
+            valueYField: legend.name,
             stacked: true,
             stroke: undefined,
-        });
+        };
+        if (legend.color) seriesSettings.fill = color(legend.color);
```

Comment:
nitpick: I added color util to chartHelper, so u can use it like `chartHelper.color()`

Reply:
wow thanks!!!!!!!